release/pipelines.release.yml

```diff
@@ -62,14 +62,24 @@ pipelines:
               JFROG_CLI_BUILD_NAME=$JFROG_CLI_BUILD_NAME
               JFROG_CLI_BUILD_NUMBER=$JFROG_CLI_BUILD_NUMBER
               JFROG_CLI_BUILD_PROJECT=$JFROG_CLI_BUILD_PROJECT
-              jf gradle clean build artifactoryPublish -x test -x functionalTest
+              ORG_GRADLE_PROJECT_signingKey=$(echo $int_mvn_central_signingKey | base64 -d)
```

Comment:
The int_mvn_central_signingKey is only available if the mvn_central integration exists in the pipeline:
```yml
integrations:
  - name: mvn_central
```

Reply:
done